noir-projects/noir-protocol-circuits/crates/rollup-lib/src/abis/previous_rollup_block_data.nr

```diff
@@ -18,8 +17,8 @@ pub struct PreviousRollupBlockData {
     pub vk_witness: MembershipWitness<VK_TREE_HEIGHT>,
 }
 
-impl Verifiable for PreviousRollupBlockData {
-    fn verify(self) {
+impl PreviousRollupBlockData {
+    fn verify(self, proof_type_id: u32) {
```

Comment:
adding an extra argument because this function is shared between the root rollup and the block merge